packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -68,14 +68,7 @@ abstract contract FundingRateApplier is FeePayer {
      *                EVENTS                *
      ****************************************/
 
-    event NewFundingRateMultiplier(
-        uint256 indexed newMultiplier,
-        uint256 lastApplicationTime,
-        uint256 applicationTime,
-        uint256 indexed paymentPeriod,
-        int256 indexed latestFundingRate,
-        int256 periodRate
-    );
+    event FundingRateUpdated(int256 newFundingRate, uint256 indexed updateTime);
```

Comment:
I wonder if we should also have an event which emits the new cumulative multiplier. I cant really forsee how that would be useful though. I imagine if some offchain infra was using events to recreate historical contract state maybe this one event is enough to do that. thoughts?

Reply:
Yeah I think its sufficient to just read the cumulative multiplier on-chain. I don't think there's a need to see historical multipliers, but you should always know the current multiplier before modifying a position.